### ISSUE TITLE
Accessibility Tools

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#7028]
Thank you for taking the time to add the important feature it was much needed, however if possible there is a need to display to the user also an Accessibility Statement/Declaration by law. We think this should be set in the  feature displayed in the tools menu.
A good idea maybe would be to allow a link to a post page on the site and thus it will be shown on that menu.

Would like to hear your  thought on the matter. Much thanks.
![Screen Shot 5782-12-25 at 19 37 37](https://user-images.githubusercontent.com/58882903/185974244-28d93620-7b9d-4061-9206-59a70769c4c1.jpg)

**Comment by DanielnetoDotCom** [comment]
Hi

what exactly is this? "Accessibility Statement/Declaration by law"

just a link with a text?

do you have any text samples?

**Comment by DanielnetoDotCom** [comment]
I mean, should this be a disclaimer link at the bottom of the tool?

I am not sure how it suppose to look like

**Comment by guymass** [comment]
Yeah, I guess a link in the bottom of the menu should do it, should be editable to allow title and url.

**Comment by awistudio** [comment]
Hello
How do I enable this option? Accessibility

**Comment by DanielnetoDotCom** [comment]
Plugin -> Layout -> enableAccessibility
